CHANGELOG.adoc

```diff
@@ -69,6 +69,7 @@ More existing APIs will be migrated to this new common pattern.
 - https://github.com/eclipse-sirius/sirius-web/issues/3630[#3630] [sirius-web] Restore support for the deletion of dangling representations
 - https://github.com/eclipse-sirius/sirius-web/issues/3666[#3666] [core] Enable EMF's intrinsicIDToEObjectMap for JSON resources to support languages which use modeled identifiers (e.g. SysMLv2).
 - https://github.com/eclipse-sirius/sirius-web/issues/3664[#3664] [core] The Default.svg image is no longer displayed with the new architecture.
+- https://github.com/eclipse-sirius/sirius-web/issues/3668[#3668] [diagram] Fix an issue that prevents wrapped label from overflow its container
```

Comment:
I think you mean something like `Fix an issue that would cause wrapped label to overflow its container`